conformance/reports/v1.3.0/kgateway/README.md

```diff
@@ -0,0 +1,31 @@
+# kgateway
+
+## Table of contents
+
+| API channel  | Implementation version                                                        | Mode    | Report                                                    |
+|--------------|-------------------------------------------------------------------------------|---------|-----------------------------------------------------------|
+| experimental | [main](https://github.com/kgateway-dev/kgateway) | default | [Link](./experimental-main-report.yaml) |
+
+## Reproduce
+
+### Steps
+
+1. Clone the kgateway repository:
+
+   ```sh
+   git clone https://github.com/kgateway-dev/kgateway.git && cd kgateway
+   ```
+
+2. Bootstrap a KinD cluster:
+
+   ```sh
+   make run
+   ```
+
+3. Run the conformance tests:
+
+   ```sh
+   make conformance
+   ```
+
+4. View and verify the conformance report: `cat _test/conformance/1.0.1-dev-report.yaml`
```

Comment:
What's the significance of 1.0.1 here?

Reply:
It's just a fake version we default in the Makefile to satisfy helm chart validation. I'll update the instructions to override that version.

Reply:
Updated in https://github.com/kubernetes-sigs/gateway-api/pull/3865/commits/5fdacc42f60595dbd4aefebbe9dc5021ed722dcc